osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearAlarmRegisterCommandExecutor.java

```diff
@@ -136,4 +166,23 @@ public AccessResultCode executeForAlarmRegister(
       throw new ConnectionException(e);
     }
   }
+
+  public Optional<AttributeAddress> findAttributeAddress(
```

Comment:
This seems to be a fairly generic function. How could we accomodate reuse?

Reply:
Can promote it to the Object Config Service.